jekyll/_cci2/vs-code-extension-quickstart.adoc

```diff
@@ -0,0 +1,117 @@
+
+Specifically, you can choose to display:
+
+* All pipelines triggered by any user (author), or only pipelines you trigger.
+* All branches, only your current branch, or a specific branch.
+** You can also choose to “Always display default branch.”
+* Only pipelines that include workflows and jobs with specific statuses.
+
+[#configure-notifications]
+== 5. Configure notifications
+
+By default, you will receive VS Code pop-up notifications every time one of the workflows in your pipelines panel enters a failing status, or needs your approval.
+
+You can configure specific workflow status changes you want to be notified for in the Settings page.
+
+[#configure-ssh-keys]
+== 6. Configure SSH keys
+
+The extension allows you to xref:ssh-access-jobs#[rerun jobs with SSH] directly from within VS Code, either in the terminal or in a link:https://code.visualstudio.com/docs/remote/ssh[remote VS Code window].
+
+. To use this feature, you need to install the link:https://marketplace.visualstudio.com/items?itemName=ms-vscode.remote-explorer[official Remote Explorer extension for VS Code].
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_ssh_remote_window.png[VS Code with remote development window]
+
+. The SSH functionality uses the same key that you use for your VCS. The easiest way to configure SSH keys for the extension is to try re-running a job with SSH, by clicking on the action icon next to the job.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_rerun_job_ssh.png[Failed job with rerun with SSH icon]
++
+The extension will try to find your key automatically, and guide you through some choices.
++
+Alternatively, you can configure SSH keys for the extension manually through the Settings page.
+
+[#open-a-config-file-to-test-language-support-features]
+== 7. Open a config file to test language support features
+
+The extension integrates a Language Server specific to CircleCI configuration files.
+
+. To verify that language support is working, open a `.yml` file in a `.circleci` directory.
++
+Even without taking any further action, you should see: 
+
+* Syntax coloration is applied. 
+* Red warnings appear if you introduce syntax errors.
+* You are able to use go-to-definition commands on the file.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_config_helper-overview-optimised.gif[Go-to definition in config]
++
+TIP: If you do not see any syntax coloration, you might be using a VS Code theme that is not compatible with our syntax highlighting.
+
+. Open the "Problems" panel to access diagnostics. The panel lists warnings and issues detected by the extension.
+
+TIP: If you do not see any of the above language support features, try running the command `CircleCI: Restart Language Server`.
```

Comment:
Can we be specific about where to run this from? I guess from the Command Pallette?

Reply:
@rosieyohannan I'd be OK to mention this for the other commands as well, @BeFunes any thoughts?

Reply:
Well, every VS Code user (even beginner) knows how to enter a command, and the majority of users don't really go to the Command Pallette but just enter Ctrl+Shift+P.  So personally I think it makes the sentence / action required sound "heavier" than it should.
That said - if you think it's better to be explicit that's absolutely fine